packages/utils/src/npm.ts

```diff
@@ -30,17 +30,25 @@ const install = (command: string) => {
     });
 };
 
+const hasYarnLock = (directory: string): Promise<boolean> => {
+    return new Promise((resolve) => {
+        fs.access(path.join(directory, 'yarn.lock'), (err) => {
+            resolve(!err);
+        });
+    });
+};
+
 /** Install the given packages. */
 export const installPackages = async (packages: string[]): Promise<boolean> => {
     /** Whether or not the package should be installed as devDependencies. */
     let isDev: boolean = false;
```

Comment:
Not technically part of your change, but I like to clean up unnecessary types when we're in the area.
```suggestion
    let isDev = false;
```